plugins/GSdx/GSRenderer.cpp

```diff
@@ -200,11 +221,12 @@ bool GSRenderer::Merge(int field)
 	{
 		if(!en[i] || !tex[i]) continue;
 
-		GSVector4i r = fr[i];
+		static std::array<GSVector4i, 2> r = { GSVector4i(0), GSVector4i(0) };
+		r[i] = DelayDisplayRectangleChange(fr[i], r[i], i);
```

Comment:
Not sure I understand your idea, this variable needs to be static as I need the previous values of the output for comparison on the function.